tests/phpunit/tests/media.php

```diff
@@ -3462,6 +3452,184 @@ public function data_wp_get_attachment_image_decoding_attr() {
 		);
 	}
 
+	/**
+	 * Test whether wp_decoding_enabled() returns the expected value.
+	 *
+	 * @ticket       58892
+	 * @dataProvider data_wp_decoding_enabled_tag_name_defaults
+	 *
+	 * @covers ::wp_decoding_enabled
```

Comment:
```suggestion
	 * @ticket 58892
	 *
	 * @covers ::wp_decoding_enabled
	 *
	 * @dataProvider data_wp_decoding_enabled_tag_name_defaults
```
Per [56070](https://core.trac.wordpress.org/ticket/56070), update the order of annotations.

Reply:
Thanks again for the reference here @mukeshpanchal27